quantum/quantum_random.py

```diff
@@ -0,0 +1,28 @@
+from projectq import MainEngine
+from projectq.ops import H, Measure
+
+
+def get_random_number(quantum_engine: projectq.cengines._main.MainEngine) -> int:
```

Comment:
Please add a doctest like all algorithmic functions in the repo...
```suggestion
def get_random_number(quantum_engine: projectq.cengines._main.MainEngine) -> int:
    """
    >>> instance(get_random_number(), int)
    True
    """
```